modules/doc/content/newsletter/2023/2023_02.md

```diff
@@ -9,7 +9,7 @@ based on libMesh's [generic projector](https://mooseframework.inl.gov/docs/doxyg
 explains all the features (siblings transfers, spatial restrictions, higher order variables, etc.) which are supported. The specific instances implemented are:
 
 - [MultiAppGeneralFieldShapeEvaluationTransfer.md]
-- [MultiAppGeneralFieldNearestNodeTransfer.md]
+- [MultiAppGeneralFieldNearestLocationTransfer.md]
```

Comment:
Thanks for updating these old newsletters. Also thanks for all the transitions to parent/child in the in-file doco and elsewhere.

Reply:
you're welcome!